crates/holochain_types/src/app/app_bundle.rs

```diff
@@ -1,3 +1,5 @@
+//! An App Bundle is an AppManifest bundled together with DNA bundles.
```

Comment:
DNA bundles or DNA manifests?

Reply:
Bundles

Reply:
sorry, new terminology for me. I'd always thought a DNA bundle was a bundle of DNAs :) I've been reading more stuff since I posted that question, and I think I get how y'all're using the word now.